src/controller/request/invoice-request.ts

```diff
@@ -54,20 +59,35 @@ export interface BaseInvoice {
 
 export interface CreateInvoiceParams extends BaseInvoice {
   byId: number,
+  street: string;
+  postalCode:string;
+  city: string;
+  country: string;
+  addressee: string,
 }
 
 /**
  * @typedef {object} CreateInvoiceRequest
```

Comment:
How so?

Reply:
Because you need to cross reference all the properties in the CreateInvoiceRequest spec, to see whether a property is included in CreateInvoiceRequest ór BaseInvoice (which is a lot of scrolling and is hard to check). If the CreateInvoiceRequest spec simply extends the BaseInvoie spec, you only need to check whether all properties in the CreateInvoiceRequest spec are part of the CreateInvoiceRequest interface and vice versa. Any other properties should then be defined in the BaseInvoice spec and interface, which you can also easily check.